docs/11-circleci/03-build.mdx

```diff
@@ -0,0 +1,683 @@
+---
+toc_max_heading_level: 5
+---
+
+# Build
+
+The Unity orb provides the
+[build job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-build) to facilitate the
+process of building your Unity project to multiple platforms.
+
+### Parameters
+
+In this sub-section, you will find information on each parameter the job provides.
+
+#### build-name
```

Comment:
I'm slightly confused by the usage of kebab-case for parameters.

In previous implementations we used camelCase, whereas CircleCI usually uses snake_case (or maybe I'm mistaken?).

It's fine but just curious if this is based on something.

Reply:
> CircleCI usually uses snake_case (or maybe I'm mistaken?).

That's correct. Parameters in CircleCI use snake_case. However, we've been using kebab for parameters in official orbs for the longest time. I don't have that decision's context, though.